reindexer/get_reindex_status.py

```diff
@@ -243,9 +249,32 @@ def main(reindex_date):
 
     print("")
 
-    print("*** Queue stats ***")
-    dlq_stats = get_dlq_stats(session_read_only, reindex_date=reindex_date)
-    non_empty_dlqs = {q_url: size for q_url, size in dlq_stats.items() if size > 0}
+    queue_stats = get_queue_stats(session_read_only, reindex_date=reindex_date)
+
+    print("*** SQS stats ***")
+    non_empty_queues = {
+        q_url: size
+        for q_url, size in queue_stats.items()
+        if not q_url.endswith("_dlq") and size > 0
+    }
+    if non_empty_queues:
+        print("The following queues still have messages:")
+        rows = [
+            [q_url.split("/")[-1].replace(f"catalogue-{reindex_date}_", ""), size]
+            for q_url, size in sorted(non_empty_queues.items())
+        ]
+        print(tabulate.tabulate(rows, tablefmt="plain", colalign=("left", "right")))
+    else:
+        print("There are no messages on queues")
```

Comment:
Here's what the new output looks like:

<img width="852" alt="Screenshot 2022-06-18 at 17 07 57" src="https://user-images.githubusercontent.com/301220/174447095-8d748a77-23a7-49b2-a11d-8de4f79f0b04.png">

This is to avoid a mistake I made on 2022-06-17; I assumed "index counts are correct" = "nothing left to do", but in fact there was a bunch of stuff on the relation embedder queues that hadn't gone through yet. Turning the pipeline down prematurely swamped the cluster and caused it to fall over.

Reply:
Excellent!  I started doing something like this once, then just started looking web interface instead.